StripeUICore/StripeUICore/Source/Controls/OneTimeCodeTextField-TextStorage.swift

```diff
@@ -55,7 +55,7 @@ extension OneTimeCodeTextField {
 
             if value.count > capacity {
                 // Truncate to capacity
-                value = value.stp_safeSubstring(to: capacity)
+                value = String(value.prefix(capacity))
```

Comment:
`stp_safeSubstring` handled negative numbers, but seems like `capacity` should never be negative, so this should be good 👍

Reply:
Yep, and I've added an `assert` to `init` to help enforce this